{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2017-1434",
    "ASSIGNER": "psirt@us.ibm.com"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "CWE-200"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/127806",
        "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/127806",
        "refsource": "MISC",
        "tags": [
          "Vendor Advisory"
        ]
      },
      {
        "url": "http://www.ibm.com/support/docview.wss?uid=swg22005740",
        "name": "http://www.ibm.com/support/docview.wss?uid=swg22005740",
        "refsource": "CONFIRM",
        "tags": [
          "Mitigation",
          "Patch",
          "Vendor Advisory"
        ]
      },
      {
        "url": "http://www.securitytracker.com/id/1039297",
        "name": "1039297",
        "refsource": "SECTRACK",
        "tags": [
          "Third Party Advisory",
          "VDB Entry"
        ]
      },
      {
        "url": "http://www.securityfocus.com/bid/100693",
        "name": "100693",
        "refsource": "BID",
        "tags": [
          "Third Party Advisory",
          "VDB Entry"
        ]
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "IBM DB2 for Linux, UNIX and Windows 11.1 (includes DB2 Connect Server) under unusual circumstances, could expose highly sensitive information in the error log to a local user."
      }
    ]
  }
}